Missionframework/modules/15_build/ui/buildDisplay_left.hpp

```diff
@@ -0,0 +1,96 @@
+
+class KPLIB_BuildCategoryList: KPGUI_PRE_Combo {
+    idc = KPLIB_IDC_BUILD_CATEGORY_LIST;
+
+    x = KP_GETCX(KP_X_VAL_LP,KP_WIDTH_VAL_LP,0,1);
+    y = KP_GETCY(KP_Y_VAL_LP,KP_HEIGHT_VAL_LP,2,40);
+    w = KP_GETW(KP_WIDTH_VAL_LP,1);
+    h = KP_GETH(KP_HEIGHT_VAL_LP,40);
+
+    onMouseZChanged = "['onMouseZChanged_BuildCategoryList', _this] call KPLIB_fnc_build_handleMouse";
+};
+
+class KPLIB_BuildSearch: KPGUI_PRE_EditBox {
+    idc = KPLIB_IDC_BUILD_SEARCH;
+
+    x = KP_GETCX(KP_X_VAL_LP,KP_WIDTH_VAL_LP,0,12);
+    y = KP_GETCY(KP_Y_VAL_LP,KP_HEIGHT_VAL_LP,3,40);
+    w = KP_GETW(KP_WIDTH_VAL_LP,(12/11));
+    h = KP_GETH(KP_HEIGHT_VAL_LP,40);
+
+    onKeyUp = "call KPLIB_fnc_build_displayFillList";
```

Comment:
```suggestion
    onKeyUp = "[] call KPLIB_fnc_build_displayFillList";
```
Concerning coding convention, no calls/spawns without parameters (or empty parameters)